gateway/src/project.rs

```diff
@@ -515,6 +518,9 @@ impl ProjectCreating {
                     ],
                     "Env": [
                         "RUST_LOG=debug",
+                        // If we don't set this, users' dependencies with a `rust-toolchain.toml`
+                        // override will compile with incompatible versions of rust.
+                        format!("RUSTUP_TOOLCHAIN={rustup_toolchain}")
```

Comment:
Since this is now in the Containerfile is should be safe to remove from here, right?

Reply:
I was sure I tried this and it didn't work, even though it obviously should, but I must have used an older image. I tested it again now and it works. Good looking out Pieter!